src/core/formatters.ts

```diff
@@ -60,7 +60,7 @@ export module Formatters {
    * @returns {Formatter} A formatter that displays exactly [precision] decimal places.
    */
   export function fixed(precision = 3) {
-    verifyPrecision(precision);
+    precision = verifyAndAdjustPrecision(precision);
```

Comment:
My personal approach if I need to verify and then possibly modify the input precision would be to have the function be called `ensure`.  In this case, I would modify the function name to be `ensurePrecision`.